lib/matplotlib/tests/test_patches.py

```diff
@@ -503,7 +503,70 @@ def test_fancyarrow_units():
 
 @image_comparison(["large_arc.svg"], style="mpl20")
 def test_large_arc():
-    ax = plt.figure().add_subplot()
-    ax.set_axis_off()
-    # A large arc that crosses the axes view limits.
-    ax.add_patch(mpatches.Arc((-100, 0), 201, 201))
+    fig, (ax1, ax2) = plt.subplots(1, 2)
+    x = 210
+    y = -2115
+    diameter = 4261
+    for ax in [ax1, ax2]:
+        a = mpatches.Arc((x, y), diameter, diameter, lw=2, color='k')
+        ax.add_patch(a)
+        ax.set_axis_off()
+        ax.set_aspect('equal')
+    # force the high accuracy case
+    ax1.set_xlim(0, 20)
+    ax1.set_ylim(0, 20)
+
+    # force the low accuracy case
+    ax2.set_xlim(-23399, 18023)
+    ax2.set_ylim(-20044, 6584)
+
+
+@image_comparison(["rotated_arcs.svg"], style="mpl20")
+def test_rotated_arcs():
+    fig, ax_arr = plt.subplots(2, 2, constrained_layout=True)
+    diameter = 3
+    th1_vec = np.linspace(0, 351, 10)
+    angle_vec = np.linspace(0, 351, 10)
+    x_vec = np.arange(10) * 10
+    y_vec = np.arange(10) * 10
+
+    for (th2, skew), ax in zip(
+        [(264, 1), (78, 1), (180, 2), (180, 0.5)], ax_arr.ravel()
+    ):
```

Comment:
Separate line saves a line and avoids terrible wrapping. (Pick a better name though.)
```suggestion
    test_vals = [(264, 1), (78, 1), (180, 2), (180, 0.5)]
    for (th2, skew), ax in zip(test_vals, ax_arr.ravel()):
```